### PR TITLE
Add Support To Honor 7A AUM-L29

### PR DESCRIPTION
**Pull request by algphello**

### CODE REVIEW

**Comment by algphello** [comment]
@phhusson

**Comment by TadiT7** [comment]
Known keys is there for a reason, to not push any values conflicting with AOSP or useless with it.

File: tests/knownKeys [hunk 1]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
 config_screenBrightnessForVrSettingDefault
 config_screenBrightnessForVrSettingMinimum
 config_screenBrightnessForVrSettingMaximum
+flingboost_param_value
```
Comment (phhusson) [inline tests/knownKeys]: doesn't exist

File: tests/knownKeys [hunk 2]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+scrollboost_param_value
```
Comment (phhusson) [inline tests/knownKeys]: doesn't exist

File: tests/knownKeys [hunk 3]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+launchboost_param_value
```
Comment (phhusson) [inline tests/knownKeys]: doesn't exist

File: tests/knownKeys [hunk 4]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+actlaunchboost_param_value
```
Comment (phhusson) [inline tests/knownKeys]: doesn't exist

File: tests/knownKeys [hunk 5]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+launchboost_packing_param_value
```
Comment (phhusson) [inline tests/knownKeys]: still doesn't exist

File: tests/knownKeys [hunk 6]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+panelview_flingboost_param_value
```
Comment (phhusson) [inline tests/knownKeys]: nope, this one doesn't exist either

File: tests/knownKeys [hunk 7]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+animationboost_param_value
```
Comment (phhusson) [inline tests/knownKeys]: nope

File: tests/knownKeys [hunk 8]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+config_enableCpuBoostForOverScrollerFling
```
Comment (phhusson) [inline tests/knownKeys]: nope

File: tests/knownKeys [hunk 9]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+config_enableCpuBoostForScroller
```
Comment (phhusson) [inline tests/knownKeys]: Hey guess what? This doesn't exist.

File: tests/knownKeys [hunk 10]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+config_enableCpuBoostForAppLaunch
```
Comment (phhusson) [inline tests/knownKeys]: no

File: tests/knownKeys [hunk 11]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+config_enableCpuBoostForActLaunch
```
Comment (phhusson) [inline tests/knownKeys]: no

File: tests/knownKeys [hunk 12]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+config_disablePacking
```
Comment (phhusson) [inline tests/knownKeys]: still not

File: tests/knownKeys [hunk 13]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+config_enableCpuBoostForPanelViewFling
```
Comment (phhusson) [inline tests/knownKeys]: aaannnddd..... nope.

File: tests/knownKeys [hunk 14]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+config_enablePerfBoostForAnimation
```
Comment (phhusson) [inline tests/knownKeys]: hum... nope.

File: tests/knownKeys [hunk 15]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+carrier_vt_available_bool
```
Comment (phhusson) [inline tests/knownKeys]: Hey!
Ah no, still not

File: tests/knownKeys [hunk 16]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+config_enableGeofenceOverlay
```
Comment (phhusson) [inline tests/knownKeys]: Please tell me what this does, it doesn't look very useful to me.

File: tests/knownKeys [hunk 17]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+launchboost_timeout_param
```
Comment (phhusson) [inline tests/knownKeys]: no

File: tests/knownKeys [hunk 18]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+actlaunchboost_timeout_param
```
Comment (phhusson) [inline tests/knownKeys]: naaa

File: tests/knownKeys [hunk 19]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+disablepacking_timeout_param
```
Comment (phhusson) [inline tests/knownKeys]: no

File: tests/knownKeys [hunk 20]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+animationboost_timeout_param
```
Comment (phhusson) [inline tests/knownKeys]: no

File: tests/knownKeys [hunk 21]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+config_wifi_supplicant_scan_interval
```
Comment (phhusson) [inline tests/knownKeys]: This doesn't sound like an hardware feature to me, just ROM's decision.
And indeed, Pixels/Nexus doesn't set this.

File: tests/knownKeys [hunk 22]
```diff
@@ -123,3 +123,25 @@ config_minimumBrightnessCurveNits
...
+config_fusedLocationProviderPackageName
```
Comment (phhusson) [inline tests/knownKeys]: I could agree with that one.

File: Huawei/msm8937/AUM/res/values/strings.xml [hunk 23]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="config_fusedLocationProviderPackageName">com.qualcomm.location</string>
+    <string name="config_ims_package">org.codeaurora.ims</string>
```
Comment (phhusson) [inline Huawei/msm8937/AUM/res/values/strings.xml]: Is this in /vendor, and working on GSI?

File: Huawei/msm8937/AUM/res/values/strings.xml [hunk 24]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="config_fusedLocationProviderPackageName">com.qualcomm.location</string>
```
Comment (phhusson) [inline Huawei/msm8937/AUM/res/values/strings.xml]: Is this in /vendor, and working on GSI?

Comment (algphello) [reply Huawei/msm8937/AUM/res/values/strings.xml]: yeap

**Pull request merged**